.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,18 @@
+name: Pull Request Labeler
+
+on:
+  pull_request:
+    types: opened
+
+jobs:
+  add_label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: add label
+        uses: actions-ecosystem/action-add-labels@v1
```

Comment:
using a (non-github) 3rd party github action is something I think we try to avoid when possible.  If you are planning on expanding on this I would suggest you check out [Github adding labels via an action](https://docs.github.com/en/actions/managing-issues-and-pull-requests/adding-labels-to-issues) and the [triage_handle_new_comments.yml](https://github.com/cypress-io/cypress/blob/develop/.github/workflows/triage_handle_new_comments.yml) workflow is an example of how to call a script in another repo where you could start out with your straight forward example and give you the flexibility to expand it down the road in a way that is much easier than having to either get a PR in the 3rd party action repo or trying to fight yml syntax and just work in normal javascript.